server/models/offer.model.js

```diff
@@ -106,10 +106,22 @@ const OfferSchema = new mongoose.Schema(
       type: String,
       required: true,
     },
-    totalAmountPayable: {
+    propertySellingPrice: {
       type: Number,
       required: true,
     },
+    paymentBreakdown: {
```

Comment:
This should be a string where the default is initial deposit
```suggestion
    paymentBreakdown: PAYMENT_BREAKDOWN_OPTIONS.INITIAL_DEPOSIT
```